frontend-dev/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "frontend-dev",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "react-scripts": "0.7.0"
+  },
+  "dependencies": {
+    "react": "^15.4.0",
+    "react-bootstrap": "^0.30.7",
+    "react-dom": "^15.4.0",
+    "react-json-table": "^0.1.1",
+    "react-router": "4.0.0-alpha.3",
+    "whatwg-fetch": "^2.0.1"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "styles": "stylus -u autoprefixer-stylus ./src/css/style.bootstrap.styl -o ./src/css/bootstrap.css",
```

Comment:
I would use cssnext/postcss without extra preprocessing. + it will do autoprefixing for you automatically.